bigbluebutton-html5/package.json

```diff
@@ -94,6 +93,7 @@
     "eslint-plugin-react": "~7.11.1",
     "husky": "^1.3.1",
     "lint-staged": "8.1.3",
+    "node-sass": "^4.14.1",
```

Comment:
Is this upgrade of node-sass required? If not, revert to old version and remove this file

Reply:
I got some errors while running html5 client notifying to update node-sass, so I updated the package.